04_node_client.asciidoc

```diff
@@ -689,15 +689,15 @@ $ docker exec lnd lncli -n regtest getinfo
 
 We now have another Lightning node running on the +lnbook+ network and communicating with bitcoind. If you are still running the c-lightning container, then there are now two nodes running. They're not yet connected to each other, but we will be connecting them to each other soon.
 
-If desired, you can run any combination of LND and c-lightning nodes on the same Lightning network. For example, to run a second LND node you would issue the +docker run+ command with a different container name like so:
```

Comment:
Sorry!  The capitalization of the original is correct because it is referring to a local instance of a Lightning network and not to the live Lightning Network.  Ignore my "correction"!